src/config/autoload/serlo.config.local.dist.php

```diff
@@ -100,7 +114,63 @@ function gaOptout(){document.cookie=disableStr+'=true; expires=Thu, 31 Dec 2099
 (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
 m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
 })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-ga('create', 'UA-20283862-4', 'serlo.org');ga('require', 'displayfeatures');ga('require', 'linkid', 'linkid.js');ga('set', 'anonymizeIp', true);ga('send', 'pageview');
+ga('create', 'UA-20283862-3', 'serlo.org');ga('require', 'displayfeatures');ga('require', 'linkid', 'linkid.js');ga('set', 'anonymizeIp', true);ga('send', 'pageview');
```

Comment:
I changed this to use the german Google Analytics id for all tenants. If we don't want that, than we need to create new ones for hindi and spanish?

Reply:
cc @SimonKoehl: we currently have different entities for de.serlo.org and en.serlo.org on GA. You mentioned a (possible) change on Slack, though. What's the plan here? If we do it like we did before, I can add entities for `hi` and `es`

Reply:
There is one more thing untill we can make a decission here, which is that I am going to test the filter functionalities on GA. I'll come back to you about this issue

Reply:
Okay, but then I probably shouldn't change the english analytics id for now?

Reply:
This is set up correctly now